lib/tasks/populate.rake

```diff
@@ -0,0 +1,23 @@
+if Rails.env.development?
```

Comment:
Any reason for using the rake task instead of seeds?

Reply:
I read an [article](https://thoughtbot.com/blog/factory_girl-for-seed-data) where the maintainer of the FactoryBot gem recommends against using it for seed data.
Instead he recommends using factories to generate development data [at the bottom of this article](https://thoughtbot.com/upcase/videos/factory-bot) because

> `db/seeds.rb` should be reserved for static data that the application requires, like the fifty US states, and be intended to run in production. This is not a good use case for our factories, which are intended to evolve over time.

Also they provide [a template for the task](https://github.com/thoughtbot/suspenders/blob/master/templates/dev.rake) that i chose to use.

Reply:
Ok, makes sense